commerce_coordinator/apps/core/signal_helpers.py

```diff
@@ -12,7 +12,7 @@ def send(self, *args, **kwargs):
         raise NotImplementedError("Coordinator Signals do not implement the send method. Use send_robust instead.")
 
 
-def coordinator_receiver(logger):
+def log_receiver(logger):
```

Comment:
renamed `coordinator_receiver` to  `log_receiver ` as `coordinator_receiver ` was confusing with [Django signal receivers](https://docs.djangoproject.com/en/4.1/topics/signals/#django.dispatch.receiver)